pcsx2/SIO/Memcard/MemoryCardFile.cpp

```diff
@@ -786,7 +786,7 @@ int FileMcd_ReIndex(uint port, uint slot, const std::string& filter)
 
 static MemoryCardFileType GetMemoryCardFileTypeFromSize(s64 size)
 {
-	if (size == (8 * MC2_MBSIZE))
+	if (size == (8 * MC2_MBSIZE) || size == _8mb/*no ecc*/)
```

Comment:
Nitpick, ideally this comment should be on the line above the if statement for readability.

Reply:
No, that comment only applies to the second part

Reply:
You can mention _8mb in the comment.